prepline_general/api/general.py

```diff
@@ -68,7 +67,7 @@ def is_expected_response_type(media_type, response_type):
     os.environ["UNSTRUCTURED_ALLOWED_MIMETYPES"] = DEFAULT_MIMETYPES
 
 
-def get_pdf_splits(pdf, split_size=1):
+def get_pdf_splits(pdf_page_splits, split_size=1):
```

Comment:
Why the rename pdf->pdf_page_splits here?

Reply:
i think i was just trying to avoid read the PDF twice with `pdf = PdfReader(file)` and then pdf is passed down as `pdf_page_splits=pdf.pages`, thats how we use the pdf in the functions, which is a list of pdf splits